src/views/VaultDetails.tsx

```diff
@@ -0,0 +1,251 @@
+              fractionDigits={USD_DECIMALS}
+            />
+          </$MutedText>
+        </$PositionContainer>
+      ),
+    },
+    {
+      key: 'vault-leverage',
+      label: stringGetter({ key: STRING_KEYS.VAULT_LEVERAGE }),
+      value: <Output value={currentLeverageMultiple} type={OutputType.Multiple} />,
+    },
+  ];
+
+  return (
+    <$VaultDetails>
+      <$Header>
+        <$WrapRow>
+          <$MarketTitle>
+            <AssetIcon symbol={id} />
+            {stringGetter({ key: STRING_KEYS.ASSET_VAULT, params: { ASSET: name } })}
+          </$MarketTitle>
+        </$WrapRow>
+
+        <$VaultDescription>
+          <p>{stringGetter({ key: STRING_KEYS.VAULT_DESCRIPTION })}</p>
+        </$VaultDescription>
+
+        <$Buttons>
+          <Link to={`${AppRoute.Trade}/${market}`}>
+            <Button
+              type={ButtonType.Button}
+              shape={ButtonShape.Pill}
+              size={ButtonSize.Small}
+              slotRight={<Icon iconName={IconName.LinkOut} />}
+            >
+              {stringGetter({ key: STRING_KEYS.VAULT_DETAILS })}
+            </Button>
+          </Link>
+          {vaultsLearnMore && (
+            <Button
+              type={ButtonType.Link}
+              shape={ButtonShape.Pill}
+              size={ButtonSize.Small}
+              action={ButtonAction.Navigation}
+              href={vaultsLearnMore}
+              slotRight={<Icon iconName={IconName.LinkOut} />}
+            >
+              {stringGetter({ key: STRING_KEYS.VAULT_FAQS })}
+            </Button>
+          )}
+        </$Buttons>
+      </$Header>
+
+      <$Header>
+        <$DepositWithdrawButtons>
+          <Button
+            type={ButtonType.Button}
+            action={ButtonAction.Primary}
+            shape={ButtonShape.Rectangle}
+            size={ButtonSize.Small}
+            slotLeft={<Icon iconName={IconName.Deposit} />}
+          >
+            {stringGetter({ key: STRING_KEYS.DEPOSIT })}
+          </Button>
+          <Button
+            type={ButtonType.Button}
+            shape={ButtonShape.Rectangle}
+            size={ButtonSize.Small}
+            slotLeft={<Icon iconName={IconName.Withdraw} />}
+          >
+            {stringGetter({ key: STRING_KEYS.WITHDRAW })}
+          </Button>
+        </$DepositWithdrawButtons>
+        <$Details items={items} withSeparators />
+      </$Header>
+    </$VaultDetails>
+  );
+};
+
+const $VaultDetails = styled.div`
+  margin: auto;
+  width: 100%;
+
+  ${layoutMixins.gridConstrainedColumns}
+  --grid-max-columns: 2;
+  --column-gap: 2.25em;
+  --column-min-width: 18rem;
+  --column-max-width: 22rem;
+  --single-column-max-width: 25rem;
+
+  justify-content: center;
+  align-items: center;
+  padding: clamp(0.5rem, 7.5%, 2.5rem);
+  row-gap: 1rem;
+
+  @media ${breakpoints.tablet} {
+    padding: 0 clamp(0.5rem, 7.5%, 2.5rem);
+  }
+`;
+const $Header = styled.header`
+  ${layoutMixins.column}
+  gap: 1.25rem;
```

Comment:
bump gap from `20px` -> `24px` on Vault details side

Reply:
I think this is fine now that details item padding is higher

Reply:
eh I adjusted it